int32[] model_ids
---
bool success
